src/quo2/components/settings/data_item/component_spec.cljs

```diff
@@ -0,0 +1,158 @@
+(ns quo2.components.settings.data-item.component-spec
+  (:require [test-helpers.component :as h]
+            [quo2.core :as quo]))
+
+(h/describe
+  "date item tests"
+  (h/test "title is visible"
+    (h/render [quo/data-item
+               {:on-press    #(js/alert (str "pressed"))
+                :blur?       false
+                :description :account
+                :icon-right? false
+                :card?       true
+                :label       :none
+                :status      :default
+                :size        :default
+                :title       "Label"
+                :subtitle    "Description"
+                :icon        :i/placeholder}])
+    (h/is-truthy (h/get-by-text "Label")))
+
+  (h/test "data item doesn't have border width if card? is false"
+    (h/render [quo/data-item
+               {:on-press    #(js/alert (str "pressed"))
+                :blur?       false
+                :description :account
+                :icon-right? false
+                :card?       false
+                :label       :none
+                :status      :default
+                :size        :default
+                :title       "Label"
+                :subtitle    "Description"
+                :icon        :i/placeholder}])
+    (h/has-style (h/query-by-label-text :data-item)
```

Comment:
hmm, I think this came up in a different discussion. I generally don't think we should be testing for styles like this. 
Mostly, testing, has it rendered, especially if different pieces causes certain parts to render. does it have the correct events needed etc

Personally I don't think we get too much value from checking in code if some styles are help (in 99% of cases)

visual tests are a better approach to this imo, however this is something we need to work towards

Reply:
As discussed, I will change the test explanation